rjs/constants.go

```diff
@@ -0,0 +1,110 @@
+package rjs
+
+import "fmt"
+
+// Generic and Constant Errors to be returned to client to maintain stability
+var (
+	ErrInternal       = fmt.Errorf("error: internal client error")
+	NoClientSet       = fmt.Errorf("no redis client is set")
```

Comment:
Prefixing this with `Err` might bring more clarity.

Reply:
Done